song-server/src/main/java/bio/overture/song/server/service/analysis/AnalysisServiceSender.java

```diff
@@ -96,8 +96,17 @@ public Analysis suppress(String studyId, String id) {
 
   /** Delegated methods */
   @Override
-  public void updateAnalysis(String studyId, String analysisId, JsonNode updateAnalysisRequest) {
-    internalAnalysisService.updateAnalysis(studyId, analysisId, updateAnalysisRequest);
+  public Analysis updateAnalysis(String studyId, String analysisId, JsonNode updateAnalysisRequest) {
+    Analysis analysis = internalAnalysisService.updateAnalysis(studyId, analysisId, updateAnalysisRequest);
+    sendAnalysisMessage(analysis, PUBLISHED, PUBLISH);
+    return analysis;
+  }
+
+  @Override
+  public Analysis patchUpdateAnalysis(String studyId, String analysisId, JsonNode patchUpdateAnalysisRequest) {
+    Analysis analysis = internalAnalysisService.patchUpdateAnalysis(studyId, analysisId, patchUpdateAnalysisRequest);
+    sendAnalysisMessage(analysis, PUBLISHED, PUBLISH);
```

Comment:
does this send the analysis in the response?

Reply:
this method returns a response. however is not used in the AnalysisController class.